packages/mobile/src/store/wallet-connect/sagas/signMessageSaga.ts

```diff
@@ -45,18 +43,21 @@ function* signMessageAsync(action: SignMessageAction) {
       break
     }
     case 'solana-phone-wallet-adapter': {
-      const signature = data
-      if (!publicKey) return
+      const { data: signature, publicKey } = action.payload
       // TODO: Refactor token-dashboard sagas #
```

Comment:
Is this todo done?

Reply:
yeah potentially, not sure of exact status with solana-phone, but will be addressed in next pr